test/pom.xml

```diff
@@ -70,6 +70,30 @@
             <groupId>org.testcontainers</groupId>
             <artifactId>testcontainers</artifactId>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-compress</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
+            <version>${commons-compress.version}</version>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-io</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
- Allow me to express my doubts about this change. Versions 1.25.0 and later of `org.apache.commons:commons-compress` actually break the public API of testcontainers-java. See https://github.com/testcontainers/testcontainers-java/issues/8338 .
- I'm guessing you actually want to control the dependency tree of jetcd-test?
```shell
<dependency>
            <groupId>io.etcd</groupId>
            <artifactId>jetcd-test</artifactId>
            <version>${jetcd.version}</version>
            <scope>test</scope>
            <exclusions>
                <exclusion>
                    <groupId>org.apache.commons</groupId>
                    <artifactId>commons-compress</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
```